.code-samples.meilisearch.yaml

```diff
@@ -857,10 +857,14 @@ settings_guide_typo_tolerance_1: |-
           "title"
         ]
       }'
-updating_guide_check_version: |-
+updating_guide_check_version_new_authorization_header: |-
   curl \
-    -X GET 'http://127.0.0.1:7700/version' \
-    -H 'Authorization: Bearer apiKey'
+      -X GET 'http://127.0.0.1:7700/version' \
+      -H 'Authorization: Bearer apiKey'
+updating_guide_check_version_old_authorization_header: |-
```

Comment:
How do you expect to us to translate that into code samples?

Because we do not support the old version anymore, and the user has no idea what is behind the usage, in other words: 

When they call a `client.index('movies')`, they do not choose what kind of header will contain `apiKey` as the value.
So I'm not sure if this is a valid code sample for the integration team to work on...

Reply:
Hi Bruno, you are correct, the integration team does not need to work on this code sample, or any other that contains the old authorization method.

Actually, we have decided that the current version of the updating guide, which describes different processes for different versions of the authorization header and dumps, is too complex. We will be [simplifying this guide](https://github.com/meilisearch/documentation/issues/1723) in the future, including removing most of the code samples marked "old".